magda-web-client/src/Components/Dataset/Add/Pages/ReviewPage/AccessAndUse.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { FunctionComponent } from "react";
+
+import { State } from "../../DatasetAddCommon";
+
+import CollapseBox from "./CollapseBox";
+import CollapseItem from "./CollapseItem";
+
+import DescriptionBox from "Components/Common/DescriptionBox";
+import * as codelists from "constants/DatasetConstants";
+import ValidationRequiredLabel from "../../ValidationRequiredLabel";
+import { shouldValidate } from "../../ValidationManager";
+import { config } from "config";
+import { getFormatIcon } from "../../../View/DistributionIcon";
+
+import "./AccessAndUse.scss";
+
+type PropsType = {
+    stateData: State;
+};
+
+const AccessAndUse: FunctionComponent<PropsType> = props => {
+    const {
+        dataset,
+        datasetPublishing,
+        licenseLevel,
+        distributions,
+        informationSecurity
+    } = props.stateData;
+
+    return (
+        <CollapseBox
+            heading="Access and use"
+            stepNum={3}
+            className="access-and-use"
+        >
+            {config.featureFlags.publishToDga ? (
+                <CollapseItem className="row" alwaysShow={true}>
+                    <div className="col-sm-3">
+                        <div className="title-box">
+                            Publish to data.gov.au as open data (*):
+                        </div>
+                    </div>
+                    <div className="col-sm-9 content-box single-line">
+                        {datasetPublishing?.publishAsOpenData?.["dga"]
+                            ? "YES"
```

Comment:
Can this be `"Yes" : "No"` so it's a bit less shouty? :)

Reply:
Updated~